src/services/groups.service.ts

```diff
@@ -0,0 +1,96 @@
+import { PrismaClient, Group } from '@prisma/client';
+import { CreateGroupDto } from '@dtos/group.dto';
+import { HttpException } from '@exceptions/HttpException';
+import { isEmpty } from '@utils/util';
+
+class GroupService {
+  public group = new PrismaClient().group;
```

Comment:
- Instantiating PrismaClient in the GroupService is not optimal 
- Consider instanstiating it once and use it everywhere in the application 
e.g 

`import { PrismaClient } from '@prisma/client';

const prisma = new PrismaClient();

class GroupService {
 public group = prisma.group;

 //.......
}`